getid3/module.tag.id3v2.php

```diff
@@ -431,7 +435,7 @@ public function Analyze() {
 				$thisfile_id3v2['minorversion_footer'] = ord($footer{4});
 			}
 			if ($thisfile_id3v2['majorversion_footer'] <= 4) {
-				$id3_flags = ord(substr($footer{5}));
+				$id3_flags = ord($footer{5});
```

Comment:
Function substr invoked with 1 parameter, 2-3 required. Was used by mistake, `$footer{5}` will be enough.